resdk/query.py

```diff
@@ -145,9 +145,9 @@ def __getitem__(self, index):
         # pylint: disable=protected-access
         if not isinstance(index, (slice,) + six.integer_types):
             raise TypeError
-        if ((not isinstance(index, slice) and index < 0) or
```

Comment:
Don't change the code, if the functionality stays the same.

Reply:
Ok, I see. You have to argue such changes better and put a description in the commit message.